### DISCUSSION TITLE
Unhandled Rejection (Error): RPC interface "PresentationRpcInterface" is not initialized.

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#573] [Q&A] [answered]
Hi.My program always reports errors:Unhandled Rejection (Error): RPC interface "PresentationRpcInterface" is not initialized.

I'm sure that "presentation RPC interface" must be initialized, and the version numbers of front end, back end and imodel bank are the same.

Then, I can't find the model in the open view, but I can use the imodel connection normally. What might be the cause of this? thank you.

**Comment by pmconne** [comment]
> I'm sure that "presentation RPC interface" must be initialized

The unhandled rejection says otherwise. What type of app do you have? Does it run the backend locally, or connect to some existing backend (which one)? Show your code (frontend, and backend if applicable) that initializes PresentationRpcInterface.

**Reply by shenshengyi** [reply]
> > I'm sure that "presentation RPC interface" must be initialized
> 
> The unhandled rejection says otherwise. What type of app do you have? Does it run the backend locally, or connect to some existing backend (which one)? Show your code (frontend, and backend if applicable) that initializes PresentationRpcInterface.

When I use frontend in simple viewer app on one computer to remotely connect to the existing backend and bank in another computer, this error is reported, and then the connection is successful. However, in the open view, I can not find the model, but I can see the model data in the attribute tree.

**Comment by pmconne** [comment]
@Josh-Schifter

**Comment by pmconne** [comment, marked as answer]
@shenshengyi reported the issue is resolved.
